fix conditional compile for DEFINED case
--- a/Src/pgp.c
+++ b/Src/pgp.c
@@ -402,7 +402,7 @@
   if (mode == PGP_ARMOR_REM || mode == PGP_ARMOR_NYMKEY || mode == PGP_ARMOR_NYMSIG)
     buf_appends(out, "Version: N/A\n");
   else
-#elif MIMIC /* end of CLOAK */
+#elif defined(MIMIC) /* end of CLOAK */
   if (mode == PGP_ARMOR_REM || mode == PGP_ARMOR_NYMKEY || mode == PGP_ARMOR_NYMSIG)
     buf_appends(out, "Version: 2.6.3i\n");
   else
